From: Johannes Schauer Marin Rodrigues <josch@debian.org>
Date: Wed, 30 Apr 2025 06:21:01 +0200
Subject: Linux 6.14

Support link_id in get_tx_power since 7a53af85d3bbdbe06cd47b81a6d99a04dc0a3963

Thanks: Andreas Beckmann <anbe@debian.org>
---
 CORE/HDD/src/wlan_hdd_cfg80211.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CORE/HDD/src/wlan_hdd_cfg80211.c b/CORE/HDD/src/wlan_hdd_cfg80211.c
index 0a0e5a1..e457702 100644
--- a/CORE/HDD/src/wlan_hdd_cfg80211.c
+++ b/CORE/HDD/src/wlan_hdd_cfg80211.c
@@ -26309,6 +26309,9 @@ static int __wlan_hdd_cfg80211_get_txpower(struct wiphy *wiphy,
 static int wlan_hdd_cfg80211_get_txpower(struct wiphy *wiphy,
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 8, 0) || defined(WITH_BACKPORTS)
 					 struct wireless_dev *wdev,
+#endif
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 14, 0)
+					 unsigned int link_id,
 #endif
 					 int *dbm)
 {
